### PR TITLE
New Time Format Cheat Sheet

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
Format all the time!

screenshot from 2016-01-24 10 02 22

---
[IA Page](https://duck.co/ia/view/unix_time_format_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/unix-time-format.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
+{
+    "id": "unix_time_format_cheat_sheet",
+    "name": "Unix Time Format Sequences",
+    "description": "Common sequences for formatting time in Unix.",
+    "aliases":["time format", "format time", "time formatting",
+               "strftime", "strptime", "linux date", "linux time",
+               "unix date"],
+    "template_type": "reference",
+    "section_order": ["Day", "Week", "Month", "Year", "Date",
+                      "Time", "Time Zone", "Misc"],
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/unix-time-format.json]: Have you got a source_url you can whack in for this?http://manpages.debian.org/cgi-bin/man.cgi?query=date would do for example

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/unix-time-format.json]: Sure - I'll just use that one.

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/unix-time-format.json]: Done.

File: share/goodie/cheat_sheets/json/unix-time-format.json [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            "key": "%G"
+        }, {
+            "val": "last two digits of year (00..99)",
+            "key": "%y"
+        }, {
+            "val": "year",
+            "key": "%Y"
+        }],
+        "Date": [{
+            "val": "locale's date and time (e.g., Thu Mar 3 23:05:25 2005",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/unix-time-format.json]: @GuiltyDolphin I think you missed ) at end

File: share/goodie/cheat_sheets/json/unix-time-format.json [hunk 3]
```diff
@@ -0,0 +1,160 @@
+{
+    "id": "unix_time_format_cheat_sheet",
+    "name": "Unix Time Format Sequences",
+    "description": "Common sequences for formatting time in Unix.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/unix-time-format.json]: @GuiltyDolphin Lets remove the period at the end

File: share/goodie/cheat_sheets/json/unix-time-format.json (hunk 2 above)
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/unix-time-format.json]: Woops! Yah, thanks @jophab

**Comment by moollaza** [comment]
@GuiltyDolphin this is looking good. Once it's on beta I'll take a quick look and merge

**Comment by GuiltyDolphin** [comment]
@moollaza Did you get a chance to have a look at this one?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=unix time format cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/unix_time_format_cheat_sheet_mobile.png)

**Comment by GuiltyDolphin** [comment]
@moollaza @mintsoft Bump - this has been on Beta quite some time with no issues. Should be good to go.

**Comment by moollaza** [comment]
@GuiltyDolphin thanks! LGTM :+1:

**Pull request merged**